packages/app/src/cli/services/environment.test.ts

```diff
@@ -129,6 +130,8 @@ const EXTENSION_A: UIExtension = {
   deployConfig: () => Promise.resolve({}),
   previewMessage: (_) => undefined,
   publishURL: (_) => Promise.resolve(''),
+  validate: () => Promise.resolve(ok({})),
+  getBundleExtensionStdinContent: () => '',
```

Comment:
Follow on:

Since we construct this object in 5 different test files we should have a test utility to construct this object.

Reply:
Issue here: https://github.com/orgs/Shopify/projects/5369/views/2